dylint/src/driver_builder.rs

```diff
@@ -168,12 +171,37 @@ fn build(opts: &crate::Dylint, toolchain: &str, driver: &Path) -> Result<()> {
     command.success()?;
 
     copy(
-        package
-            .join("target")
-            .join("debug")
-            .join(format!("dylint_driver-{}", toolchain)),
+        package.join("target").join("debug").join(format!(
+            "dylint_driver-{}{}",
+            toolchain,
+            consts::EXE_SUFFIX
+        )),
         driver,
     )?;
 
+    // To succesfully determine the dylint driver Version on Windows,
```

Comment:
```suggestion
    // MinerSebas: To succesfully determine the dylint driver Version on Windows,
```
I find it helpful to know who left comments. (I have yet to add a CONTRIBUTING.md file.)